src/components/Collection/Tmdb/AniDBEpisode.tsx

```diff
@@ -34,7 +34,7 @@ const AniDBEpisode = React.memo(({ episode, extra, isOdd, onIconClick }: Props)
       data-tooltip-content={episode.Name}
     >
       <div className="line-clamp-1 text-xs font-semibold opacity-65">
-        {episode.AniDB?.AirDate ?? 'Airdate Unknown'}
+        {episode.AniDB?.AirDate ?? '????-??-??'}
```

Comment:
Is that more readable then "Unknown Airdate"?

Reply:
This will confuse people.

Reply:
> Is that more readable than "Unknown Airdate"?

Please read the sauce code again — it says **Airdate Unknown**, not **Unknown air-date**. Impo, I found it a more coherent experience to display all "?"s in the same width and size as the formatted date, rather than using an improperly formed sentence with one made-up word ("airdate" instead of "air-date") and one proper word, written in reverse order, putting the descriptive term after the word it describes.

It was triggering my OCD _so_ **hard** that I submitted this PR. But then again, feel free to not merge this if you disagree with the change.

Reply:
Made up?

https://www.merriam-webster.com/dictionary/airdate

Reply:
You can change it to fix the order, but not '?'

Reply:
Bunch of question marks will lead to people report that something is broken, you can't show information like that to users.

Reply:
> Made up?
> 
> https://www.merriam-webster.com/dictionary/airdate

Both my phone and pc doesn't recognise it as a word from it's included dictionary. I'll leave it at that.

Reply:
> You can change it to fix the order, but not '?'

any suggestions?

Reply:
Unknown airdate

Reply:
> > Made up?
> > https://www.merriam-webster.com/dictionary/airdate
> 
> Both my phone and pc doesn't recognise it as a word from it's included dictionary. I'll leave it at that.

That's like saying you'll change "color" to "colour" because it's not recognized. I did check it in Google before putting "Airdate" there because I had the same doubt.